prediction_market_agent/db/models.py

```diff
@@ -11,3 +11,21 @@ class LongTermMemories(SQLModel, table=True):
     task_description: str
     metadata_: Optional[str] = None
     datetime_: datetime
+
+
+PROMPT_DEFAULT_SESSION_IDENTIFIER = "microchain-streamlit"
+
+
+class Prompt(SQLModel, table=True):
+    """Checkpoint for general agent's prompts, as a way to restore its past progress."""
+
+    __tablename__ = "prompts"
+    __table_args__ = {
+        "extend_existing": True
+    }  # required if initializing an existing table
+    id: Optional[int] = Field(default=None, primary_key=True)
+    prompt: str
+    # This allows for future distinction between user sessions, if prompts from a specific
+    # user (or app) should be persisted.
+    session_identifier: Optional[str] = Field(default=PROMPT_DEFAULT_SESSION_IDENTIFIER)
```

Comment:
```suggestion
    session_identifier: str
```

This default value is specified on multiple places, that becomes error prone after a while

Reply:
Sure, agree with the reasoning of maintainability - added default values one step before writing/reading from DB instead of defining the default value on a DB level. Hope that's fine now.